test/starknet/shared/short.ts

```diff
@@ -0,0 +1,80 @@
+export const SHORT_STR_SIZE = 31;
```

Comment:
Could you use these functions from sx.js, I've moved the code there, you can do this like this: https://github.com/snapshot-labs/sx-api/blob/master/src/writer.ts#L1

Reply:
[Add sx-api dep; Remove setupcalldata function](https://github.com/snapshot-labs/sx-core/pull/54/commits/fecb1fc3d7d126c81e82b9063706002c758b0230)